x-pack/plugins/security_solution/public/cases/components/create/sync_alerts_toggle.test.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { act, waitFor } from '@testing-library/react';
+
+import { useForm, Form, FormHook } from '../../../shared_imports';
+import { SyncAlertsToggle } from './sync_alerts_toggle';
+import { schema, FormProps } from './schema';
+
+describe('SyncAlertsToggle', () => {
+  let globalForm: FormHook;
+
+  const MockHookWrapperComponent: React.FC = ({ children }) => {
+    const { form } = useForm<FormProps>({
+      defaultValue: { syncAlerts: true },
+      schema: {
+        syncAlerts: schema.syncAlerts,
+      },
+    });
+
+    globalForm = form;
+
+    return <Form form={form}>{children}</Form>;
+  };
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('it renders', async () => {
+    const wrapper = mount(
+      <MockHookWrapperComponent>
+        <SyncAlertsToggle isLoading={false} />
+      </MockHookWrapperComponent>
+    );
+
+    expect(wrapper.find(`[data-test-subj="caseSyncAlerts"]`).exists()).toBeTruthy();
+  });
+
+  it('it toggles the switch', async () => {
+    const wrapper = mount(
+      <MockHookWrapperComponent>
+        <SyncAlertsToggle isLoading={false} />
+      </MockHookWrapperComponent>
+    );
+
+    act(() => {
+      wrapper.find('[data-test-subj="caseSyncAlerts"] button').first().simulate('click');
+      wrapper.update();
+    });
```

Comment:
you do not need the `act` or `update`
```suggestion
    wrapper.find('[data-test-subj="caseSyncAlerts"] button').first().simulate('click');
```